ts/src/bitfinex2.ts

```diff
@@ -765,6 +750,30 @@ export default class bitfinex2 extends Exchange {
                     };
                 }
             }
+            result[code] = {
+                'id': fid,
+                'uppercaseId': id,
+                'code': code,
+                'info': [ id, label, pool, feeValues, undl ],
+                'type': type,
+                'name': name,
+                'active': true,
```

Comment:
Why do we set this as true if isDepositEnabled and isWithdrawEnabled are false, while other exchanges disable the currency?